### PR TITLE
Add Translation Files of Three Chinese Variants

### PR DESCRIPTION
**Pull request by tszhangchan**
Add translation files of Chinese of Mainland China, Hong Kong SAR and Taiwan respectively.

`zh_cn`: Simplified Chinese (Mainland China)
`zh_hk`: Traditional Chinese (Hong Kong SAR)
`zh_tw`: Traditional Chinese (Taiwan)

### CODE REVIEW

File: common/src/main/resources/assets/backupmanager/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "backupmanager:gui.backups.title" : "备份列表",
+  "backupmanager:gui.backups.created" : "备份时间: %s",
+  "backupmanager:gui.backups.provider" : "备份来源: %s",
+  "backupmanager:gui.backups.restore_name" : "请输入世界名，选择世界页面将显示该名称",
+  "backupmanager:gui.backups.error_occurred" : "错误！\n\n%s",
+  "backupmanager:gui.backups.restored" : "成功重载备份",
```
Comment (zly2006) [inline common/src/main/resources/assets/backupmanager/lang/zh_cn.json]:
```suggestion
  "backupmanager:gui.backups.restored" : "成功还原备份",
```

File: common/src/main/resources/assets/backupmanager/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "backupmanager:gui.backups.title" : "备份列表",
+  "backupmanager:gui.backups.created" : "备份时间: %s",
+  "backupmanager:gui.backups.provider" : "备份来源: %s",
+  "backupmanager:gui.backups.restore_name" : "请输入世界名，选择世界页面将显示该名称",
```
Comment (zly2006) [inline common/src/main/resources/assets/backupmanager/lang/zh_cn.json:5]:
```suggestion
  "backupmanager:gui.backups.restore_name" : "请输入世界名，所有存档数据将会还原至此文件夹，该名称将会在选择世界页面显示",
```

**Review by zly2006 — COMMENTED** [review]
另：香港、台湾语言习惯和大陆有诸多不同，所以 **不要使用繁简转换**

请找一个AI大模型辅助翻译

**Comment by tszhangchan** [comment]
但是，根據原文來看，`backupmanager:gui.backups.restore_name` 的原有翻譯是否會更準確？

> Please provide a world name, The restored world will show up with this name in world selection screen.

> 请输入世界名，选择世界页面将显示该名称。

**Comment by zly2006** [comment]
> 但是，根據原文來看，`backupmanager:gui.backups.restore_name` 的原有翻譯是否會更準確？
> 
> > Please provide a world name, The restored world will show up with this name in world selection screen.
> > 
> > 请输入世界名，选择世界页面将显示该名称。

我只是建议，我觉得可以修改一下更方便理解。主要看你的意思来即可。最近我在写备份模组，直接从这边抄了个gui走（笑

**Comment by gigabit101** [comment]
command_music will need to sign the CLA before we can approve this

**Comment by tszhangchan** [comment]
> command_music will need to sign the CLA before we can approve this

That is also me

And I forgot to change my username before I make a `git commit` and `git push`

What should or can I do

**Comment by ThePaul-T** [comment]
I believe you could force-push over your previous commits on your fork and then it should solve it? My git knowledge isn't fantastic unfortunately.

**Comment by tszhangchan** [comment]
Wait I haven't recommit my new changes
https://github.com/tszhangchan/BackupManager/commit/a9d9254fbc5d44e403b82aa2b1a52888f79283f3

**Comment by gigabit101** [comment]
We have not released this yet so feel free to make more changes in a new PR

**Pull request merged**